test/integration/wdio.conf.js

```diff
@@ -392,7 +387,6 @@ const wdioConfig = {
     // Services take over a specific job you don't want to take care of. They enhance
     // your test setup with almost no effort. Unlike plugins, they don't add new
     // commands. Instead, they hook themselves up into the test process.
-    services: [[testParams.SERVICE, serviceConfigs[testParams.SERVICE]]],
```

Comment:
These told wdio to use the chromedriver service, which doesn't exist anymore, and was crashing.